src/AnalyticGeometry/Vector.h

```diff
@@ -2,8 +2,8 @@
 
 #define VECTOR_H
 
-class B_IDataFile;
-class B_ODataFile;
+#include "IDataFile.h"
+#include "ODataFile.h"
```

Comment:
Otherwise it's not working. I presume because these are defined in `Vector.cpp`:
```
/*
* Module:                 Blade.exe
* Entry point:            0x0040ACDD
* VC++ mangling:          ??5@YAAAVB_IDataFile@@AAV0@AAVB_Vector@@@Z
*/

B_IDataFile& operator >>(B_IDataFile& file, B_Vector &vector)
{
    file.Read(&vector, sizeof(B_Vector));
    return file;
}


/*
* Module:                 Blade.exe
* Entry point:            0x0040ACF4
* VC++ mangling:          ??6@YAAAVB_ODataFile@@AAV0@AAVB_Vector@@@Z
*/

B_ODataFile& operator <<(B_ODataFile& file, B_Vector &vector)
{
    file.Write(&vector, sizeof(B_Vector));
    return file;
}
```
They're being imported from the `BBLibc` instead

Reply:
OK

Reply:
I think we might move it to `IDataFile.cpp` file

Reply:
No, they originally belongs to `Vector.cpp`(`Blade.exe`). `IDataFile.cpp` is at `BBLibc.dll`